lm/srilm.py

```diff
@@ -353,7 +353,7 @@ def _create_cmd(self) -> List[str]:
     def run(self):
         """Call the srilm script and extracts the different weights from the log, then relinks log to output folder"""
         cmd = self._create_cmd()
-        subprocess.check_call(cmd, stdout=open("cbm.log", "wb"), stderr=subprocess.STDOUT)
+        subprocess.check_call(cmd, stdout=open("cbm.log", "wt"), stderr=subprocess.STDOUT)
```

Comment:
I don't understand this change. It should be binary, or not?

Reply:
Should it? This is just a textfile i think.

Reply:
It does not matter what it is. It matters whether you get a binary or text stream from the `subprocess` module, and I'm pretty sure by default you get a binary stream (over `bytes`, not `str`). So you store it in binary.